test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Telemetry/NuGetTelemetryServiceTests.cs

```diff
@@ -64,6 +66,15 @@ public void NuGetTelemetryService_EmitProjectInformation(NuGetProjectType projec
             Assert.NotNull(isPRUpgradable);
             Assert.IsType<bool>(isPRUpgradable);
             Assert.Equal(projectInformation.IsProjectPRUpgradable, isPRUpgradable);
+
+            var fullPath = lastTelemetryEvent
+                .GetPiiData()
+                .Where(kv => kv.Key == nameof(ProjectTelemetryEvent.FullPath))
+                .First()
+                .Value;
+            Assert.NotNull(fullPath);
+            Assert.IsType<string>(fullPath);
+            Assert.Equal(projectInformation.FullPath, fullPath);
```

Comment:
`projectInformation.FullPath`'s implementation is the same as how you get `fullPath` from `lastTelemetryEvent`. This means you're effectively asserting that `true == true`. This assertion can't catch any regressions.

Reply:
Fixed.